### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Activities_Activities tab_ScreenReader: When the voiceover is turned on and the user uses Control + Option + Arrow keys on successful message notification, the focus is moving to 'tick' mark decorative image.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5769]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Navigate to any of the file inside "Blob Containers"
6. Tab to 'Activities' tab.
7. Tab till any of the successfully completed message.
8. Turn on voiceover and use VO + Arrow keys (Control + Option referenced as VO)

## Actual:
When the voiceover is turned on and the user uses Control + Option + Arrow keys on successful message notification, the focus is moving to 'tick' mark decorative image.

## Expected:
When the voiceover is turned on and the user uses Control + Option + Arrow keys on successful message notification, the focus should not move to 'tick' mark decorative image. Decorative image should have null alt text (alt="") so that can be ignored by assistive technologies such as screen readers.

## User Impact:
Screen reader users may not be able to understand the purpose of the image when used for decorative purpose.

## Attachment:
![13_Focus goin to](https://user-images.githubusercontent.com/98393966/169970148-0d087c78-1d3e-4d75-a1df-7b8fdd651836.gif)

**Comment by MRayermannMSFT** [comment]
@bbarik06 can you clarify what you would expect voice over to read as you press the various possible arrow keys, starting from voice over focusing "Clear completed".

So for example, in your reply I am looking for something like:

2. "Clear completed" focused
3. ctrl + option + right arrow
4. "Clear successful" focused
5. ctrl + option + right arrow
6. etc.
7. etc.
8. etc.

**Comment by bbarik06** [comment]
I would suggest you not to bring focus to the 'Decorative image' since the information that you want to attach with it such as 'Clear successful' is already present with the text message beside it in the form of text 'completed'. From accessibility point of view, focus should not move decorative image.

**Comment by craxal** [comment]
The green checkmark is not decorative. It is used to convey the state of the activity (success), so we need to see if we have an appropriate alt text.

**Comment by Shivamd1** [comment]
@craxal  this issue is specifically to screen reader users, if you provide alt text to the image then the announcement will be redundant, as the same text is available beside the image. So we will request you to keep the alt-text blank and not to have focus move to the tick image.

**Comment by craxal** [comment]
@Shivamd1 Activities have a "status" property as well as a "message" property. The image reflects the value of the "status" property, which is independent of the "message" value. Therefore, it can, in theory, convey information that is not in "message". Therefore, this information should not be discarded for screen reader users. Visual users can quickly verify the status of an activity by looking at the icon. From a usability standpoint, screen reader users would not get this same benefit if it was treated as purely decorative.

To complicate things, even if we were to set `alt=""`, the image is not in an `<img>` element, and therefore there is no `alt` attribute for us to modify. It is an inline SVG, and the standard way for providing alternative text is to include a `<title>` element. But we seem to have conflicting requirements for screen reader + mouse users in that regard (see #3506).

**Comment by bbarik06** [comment]
@craxal Please join office hour call for further clarification:  https://outlook.office365.com/owa/calendar/microsoft2779007@microsoft.onmicrosoft.com/bookings/s/liYlILD8qUK6LE24eXV-Vg2

**Comment by MRayermannMSFT** [comment]
Hi @bbarik06

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-darwin-x64.zip
